content/blog/2025-11-11-implementing-olympia-part-0/index.md

```diff
@@ -0,0 +1,196 @@
+
+This architecture transforms Ethereum Classic from a **donor-dependent network** into a **self-funding, self-governing public utility** — powered by on-chain activity and aligned incentives.  
+(ECIP-1115 extends this loop by optionally smoothing miner incentives at the governance layer, without altering the consensus-level foundation.)
+
+---
+
+## Pioneering PoW: A First Among Proof-of-Work Networks
+
+While most PoW chains rely on centralized foundations or off-chain grants, **Ethereum Classic can now finance its own evolution through protocol logic**.
+
+Olympia demonstrates what only a **smart-contract-capable Proof-of-Work network** can achieve — a sustainable economic loop that rewards security, development, and utility, **without inflation or trusted intermediaries**.
+
+It preserves the values that defined ETC in 2016 — **immutability, Proof-of-Work, and credible neutrality** — while finally addressing the long-standing challenges of decentralized, durable funding and long-term network security.
+
+With **ECIP-1115**, ETC can optionally extend this sustainability loop into miner incentives themselves, providing a mechanism for **predictable, non-inflationary security budgeting.**
+
+---
+
+## The Road Ahead
+
+This series, **Implementing Olympia: From Draft to Testnet**, will chronicle the transition from concept to code.  
+Over the coming months, we’ll explore:
+
+- The engineering of the **ECIP-1111 BASEFEE redirection logic**  
+- The design and audit of the **ECIP-1112 Treasury Contract**  
+- The structure and deployment of the **ECIP-1113 DAO Framework**  
+- The workflow and transparency of the **ECIP-1114 Funding Process**  
+- The optional **ECIP-1115 smoothing mechanism** and how governance can configure long-term miner-reward curves without modifying consensus  
+
+Together, these explorations document how Olympia moves from draft to deployed code — setting the stage for **mainnet activation**.
+
+Each installment will take readers deeper into the contracts, logic, and governance systems now moving toward ETC’s testnets — showing how Olympia is being implemented to be **auditable, adaptable, and permanently decentralized.**
+
+---
+
+## A Bright Future: The Olympia Implementation Era Begins
+
+**Olympia Day — 11 / 11 / 2025** marks the moment Ethereum Classic begins implementing the Olympia upgrade suite.  
+Olympia is not a single hard fork; it’s the start of a **living framework** where on-chain activity directly powers network longevity.
+
+For miners, developers, ecosystem sponsors, and long-time community members alike, this marks a return to Ethereum Classic’s founding ethos:
+
+> “A credibly neutral, self-sustaining Proof-of-Work network built to last — secured by miners, maintained by builders, supported by institutions, and enforced by code.”
+
+---
+
+## Next in the Series
+
+👉 **Part 1 — Implementing ECIP-1111: BASEFEE Redirection and Miner Alignment**
+
+💚⚒️  
+*Ethereum Classic Core Devs*
```

Comment:
@chris-mercer who are the core devs you are referencing here, and did they sign off on this?